### PR TITLE
Create recommendation from hsieh for CA

### LINKED ISSUE #6: create a new file for Hsieh's restaurant
## SF, CA
- [x] Create a branch
- [x] Add commit (make changes)
- [x] Open a pull request
- [x] Collaboration
- [x] Seek approval
- [x] Merge

### PR DESCRIPTION
**Pull request by angelaRWDS**
closes #6

### CODE REVIEW

File: places/test/hsiehf3-SF.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# SF, CA
+### Places to Eat
+- Blue Bottle
```
Comment (githubteacher) [inline places/test/hsiehf3-SF.md:3]: I hear this place is great! Do you have recommendations for 🍵 ?

Comment (angelaRWDS) [reply places/test/hsiehf3-SF.md:3]: Yes! I like Asha tea house.

File: places/test/hsiehf3-SF.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+- Tartin
```
Comment (leey46) [inline places/test/hsiehf3-SF.md:4]: What is this place for?

Comment (angelaRWDS) [reply places/test/hsiehf3-SF.md:4]: It's a bakery!

**Review by leey46 — APPROVED** [review]

**Comment by angelaRWDS** [comment]
👍 This looks good to go.

**Pull request merged**